src/jit/simd.cpp

```diff
@@ -1156,6 +1156,203 @@ SIMDIntrinsicID Compiler::impSIMDRelOp(SIMDIntrinsicID      relOpIntrinsicId,
 #endif // !_TARGET_XARCH_
 }
 
+//---------------------------------------------------------------------
+// impSIMDVectorZero: create a GT_SIMD node for Vector<T>.Zero
+//
+// Arguments:
+//    simdType  -  simd vector type
+//    baseType  -  element type of vector
+//    size      -  size of vector in bytes
+GenTreePtr Compiler::impSIMDVectorZero(var_types simdType, var_types baseType, unsigned size)
+{
```

Comment:
I have considered in the past that it would be useful to have a method to create a SIMD zero node, but it would seem to make sense as `gtNewSIMDZero` or something, as it's not necessarily restricted to the importer.